triggers/test/daml/Tests/Matching.daml

```diff
@@ -90,9 +101,170 @@ runMatchingTest parties@Parties{userAdmin = operator; ..} = do
             side = Order.Buy
             optExchangeFee = None
             timeInForce = Order.GTC
+            orderType = Order.Limit with price = 150.0
+
+  debug "Waiting for orders to be executed..."
```

Comment:
Does running this test with the Exberry integration still pass with these additions? The test was initially intended to pass with both the matching engine and the integration - though had it been this detailed originally, we probably would have caught the matching engine issue fixed in this PR earlier!